Inventory.java

```diff
@@ -0,0 +1,45 @@
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+/*
+*  Inventory.java sets the carSeatCovers list and adds a car into the list. Creates a list for themes.
+* method for comparing the user entry to the products in the inventory.txt file */
+
+public class Inventory {
+
+    // field
+    private final List<CarSeatCover> carSeatCovers = new ArrayList<>();
+
+    // method to add a car seat to the above field
+    public void addCarSeatCover(CarSeatCover carseatcover) {this.carSeatCovers.add(carseatcover);}
```

Comment:
consider getting the intellij code formatting to work if you want to show this coursework off in your github :) 

code that is well indented and formatted is something interviewers will look for